doc/nrf/installation/updating.rst

```diff
@@ -17,82 +17,78 @@ However, if you work with a specific release of the |NCS|, you do not need to up
+   .. group-tab:: nRF Connect for VS Code
 
-Updating in Toolchain Manager
-=============================
+      The |nRFVSC| lets you update the associated |NCS| repositories within the :guilabel:`Source Control View`.
+      For detailed instructions, see the `west module management`_ page in the extension's documentation.
 
-.. note::
-   The SDK versions available in Toolchain Manager are for specific releases.
-   Updating the SDK repositories in Toolchain Manager might therefore be required only in exceptional situations.
+      You can also change the SDK or toolchain in the |nRFVSC| to a new one.
+      Complete the steps listed on the `How to change SDK and toolchain versions`_ page in the extension's documentation.
 
-If you installed the |NCS| automatically using the :ref:`Toolchain Manager <auto_installation>`, complete the following steps to update the repositories in Toolchain Manager:
+   .. group-tab:: Command line
 
-1. Open the Toolchain Manager application in nRF Connect for Desktop.
-#. Click the button with the arrow pointing down next to the installed |NCS| version to expand the drop-down menu with options.
+      To manage the ``nrf`` repository (the manifest repository) from command line, use Git.
+      To make sure that you have the latest changes, run ``git fetch origin`` to :ref:`fetch the latest code <dm-wf-update-ncs>` from the `sdk-nrf`_ repository.
+      Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
+      Running ``west update`` updates the project repositories to the state specified in this manifest file.
 
-   .. figure:: images/gs-assistant_tm_dropdown.png
-      :alt: The Toolchain Manager dropdown menu for the installed nRF Connect SDK version, cropped
+      .. include:: installing.rst
+         :start-after: west-error-start
+         :end-before: west-error-end
 
-      The Toolchain Manager dropdown menu options
+      Switching to release example
 
-#. In the drop-down menu, click :guilabel:`Update SDK`.
+         .. toggle::
 
-Updating from command line
-==========================
+            To switch to release |release| of the |NCS|, enter the following commands in the ``ncs/nrf`` directory:
 
-To manage the ``nrf`` repository (the manifest repository) from command line, use Git.
-To make sure that you have the latest changes, run ``git fetch origin`` to :ref:`fetch the latest code <dm-wf-update-ncs>` from the `sdk-nrf`_ repository.
-Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
-Running ``west update`` updates the project repositories to the state specified in this manifest file.
+            .. parsed-literal::
+               :class: highlight
 
-.. include:: installing.rst
-   :start-after: west-error-start
-   :end-before: west-error-end
+               git fetch origin
+               git checkout |release|
+               west update
```

Comment:
This seems like it should be in the part earlier where I left a comment

Reply:
Tried to rephrase the headers for clarity.